common/app/model/liveblog/BlockElement.scala

```diff
@@ -176,6 +177,7 @@ object BlockElement {
       case Form            => Some(FormBlockElement(None))
 
       case EnumUnknownElementType(f) => Some(UnknownBlockElement(None))
+      case Callout                   => Some(UnsupportedBlockElement(None))
 
     }
   }
```

Comment:
This pattern match is in a make method that's converting capi element to a block element. In composer currently, callout can only be created as a body element in articles rather than a block elemenmt in liveblogs. So we don't cover for `callout` case when dealing with liveblog and as a result return a `UnsupportedBlockElement` given a callout element